### PR TITLE
Add basic styling rules

### PR DESCRIPTION
**Pull request by CathrinTruchan**
I added some basic styling rules to the GlobalStyles.js and implemented them in all my StyledComponents. Also I changed the positioning of the elements in the AsanaCard from flexbox to grid.

### CODE REVIEW

File: pages/flow/index.js [hunk 1]
```diff
@@ -133,9 +134,8 @@ const StyledContainer = styled.section`
...
 const StyledCloseButton = styled.button`
   border: none;
-  background-color: #5d6bea;
-  color: #f5f5f5;
-  padding: 0;
+  background-color: var(--background-neutral);
+  color: var(--primary);
   width: 1.5rem;
   height: 1.5rem;
   border-radius: 50%;
```
Comment (PhilippWeiss85) [inline pages/flow/index.js:142]: Are you sure you want to style the border radius in percentage?

Comment (CathrinTruchan) [reply pages/flow/index.js:142]: It's a circle. Can it cause any problems?

File: components/GlobalStyle.js [hunk 2]
```diff
@@ -17,26 +34,63 @@ html {
...
+    font-size: 1.25rem;
+    color: var(--text-dark);
+    text-align: center;
+    margin: 1rem 0;
+    font-family: "Montserrat Alternates", "Open Sans", Helvetica, Arial, sans-serif;
+  }
   
-      * {
-          box-sizing: border-box;
-      }
```
Comment (Kheirah) [inline components/GlobalStyle.js:34]: Did you mean to delete the border-box or did you removed it by accident? Just curious. Border box usually simplifies the sizing of elements.

File: pages/flow/index.js (hunk 1 above)
Comment (PhilippWeiss85) [reply pages/flow/index.js:142]: Oh no sorry. Did not look at your vercel preview. Its of course perfect for a circle

**Review by PhilippWeiss85 — APPROVED** [review]
lgtm!

**Review by Kheirah — APPROVED** [review]
Looks good! Just have a look at the border box you removed in case you accidentally deleted it.

Keep up the good work!

File: components/GlobalStyle.js (hunk 2 above)
Comment (CathrinTruchan) [reply components/GlobalStyle.js:34]: Hi Ernst, thanks for your feedback! I accidentally had the border-box rule twice so I deleted it once ;)

File: pages/flow/index.js (hunk 1 above)
Comment (CathrinTruchan) [reply pages/flow/index.js:142]: Thanks for the hint! I will check how it scales in the browser :)

**Pull request merged**